### DISCUSSION TITLE
Nintendo 3DS port

### DISCUSSION THREAD

**Discussion by kironia** [eltikia/openroguelike#6] [Q&A] [answered]
What would you say about porting this game to 3DS?

**Comment by Stagyrite** [comment, marked as answer]
A port of an early version of the openroguelike to 3DS is no longer available, but here are two photos of a 3DS console running [dsband](https://github.com/Stagyrite/openroguelike/tree/dsband "Stagyrite/openroguelike at dsband").

![dsband-1](https://github.com/eltikia/openroguelike/assets/55529808/cf70bae3-eb02-4404-9d8b-1c94b28668c1)

![dsband-2](https://github.com/eltikia/openroguelike/assets/55529808/df5977dc-5e46-4600-9e4d-02c159a4a61d)
